rtree.go

```diff
@@ -765,7 +765,8 @@ func (tree *Rtree) nearestNeighbor(p Point, n *node, d float64, nearest Spatial)
 
 		for _, e := range n.entries {
 			minDist := p.minDist(e.bb)
-			if minDist > minMinMaxDist {
+			// Add a bit of tolerance to guard against floating point rounding errors.
+			if minDist > minMinMaxDist+1e-6 {
```

Comment:
Could you extract this to a named constant?

Reply:
Yes, of course, I have extracted `FloatingPointTolerance` into a named number.